.github/workflows/pull-request.yml

```
@@ -253,6 +253,39 @@
+            - name: Run Docker Container
+                run: |
+                  docker run -d --name talawa-admin-app-container -p 8080:8080 talawa-admin-app
+
+            - name: Check if Talawa Admin app starts in Docker
+                runs-on: ubuntu-latest
+                needs: [Code-Quality-Checks]
+                steps:
+                  - name: Checkout the Repository
+                    uses: actions/checkout@v4
+        
+        - name: Stop Docker Container 
```

yamllint

[error] trailing spaces

(trailing-spaces)